win2xcur/multiscale.py

```diff
@@ -0,0 +1,32 @@
+from win2xcur.cursor import CursorFrame, CursorImage
+
+MULTSCALE = [16, 24, 32, 48, 64, 96, 128, 192, 256]
+
+def generates_frames(cursor, min_size: int) -> None:
+    """Generates multiple sizes for each cursor.
+
+    Args:
+        cursor (Cursor): The cursor to generate sizes for.
+        min_size (int): The minimum size to generate.
+
+    Returns:
+        List[Cursor]: The generated cursors.
+    """
+    frames = cursor.frames
+    new_frames = []
+    image_size = frames[0].images[0].nominal
```

Comment:
Shouldn't you pick the biggest image in the frame if you are resizing it? I am pretty sure the code doesn't sort it from biggest to smallest.